keras/src/layers/preprocessing/image_preprocessing/mix_up_test.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import pytest
+
+from keras.src import layers
+from keras.src import testing
+
+
+class MixUpTest(testing.TestCase):
```

Comment:
Please add a test for tf.data compatibility (like what we have for the other augmentation layers)

Reply:
I've added tf_compatiblity test.